frontend/src/components/landing/DemoPhone.module.scss

```diff
@@ -19,6 +19,11 @@
     left: 50%;
     transform: translateX(-50%);
 
+    &.phone {
```

Comment:
Required change by dropping SVG in favor of PNG format. 

Optimization note: the PNGs were smaller than the SVGs specifically bc the SVGs had base64/image blobs INSIDE them. Working with UX to revise images to match current UI and will have more vector (non-embedded static) graphics for the new ones.